DEPLOY_NETLIFY.md

```diff
@@ -0,0 +1,36 @@
+# Via Command Line
+
+```sh
+npm install -g netlify-cli
+
+netlify login
+netlify init -m
+```
+
+Follow de instructions as well as example below:
+![image](./.assets/netlify/step1_cli.png)
+
+- In `Give this Netlify SSH public key access to your repository`, needed add deploy key in repository (https://github.com/natura-cosmeticos/natds/settings/keys)
```

Comment:
Change `needed add deploy key in repository` to `add the deploy key in the repository`